thorlcr/graph/thgraphmaster.cpp

```diff
@@ -2802,6 +2802,8 @@ void CMasterGraph::getFinalProgress(bool aborting)
         {
             unsigned slave;
             msg.read(slave);
+            if (0 == slave) // worker no longer had graph registered
```

Comment:
Is slave 0 or 1 based?  If the corresponsing code is in slavmain it seems to be 0 based.

And/or doesn't the change in slavmain mean this will never be 0?

Reply:
it is 0 based, this guard is wrong, will fix.
The main fix is waiting when aborting, so this guard is now unlikely to be used/hit, but it's wrong.